easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-foss-2020a.eb

```diff
@@ -0,0 +1,29 @@
+name = 'HDF5'
+version = '1.10.1'
```

Comment:
Why this old bugfix version?
I know there are ABI differences between 1.12 and 1.10 series, but there is already a 1.10.6 version in 2020a.

Also, if this is truly needed, it should be moved down to gompi level.